unit/src/MooseEnumTest.C

```diff
@@ -130,6 +130,10 @@ MooseEnumTest::testErrors()
     std::string msg(e.what());
     CPPUNIT_ASSERT( msg.find("Invalid option") != std::string::npos );
   }
+  catch(...)
+  {
+    // even though we throw std::runtime_exception, we are not able to catch it here on Mac OS X (it has some other type)
+  }
```

Comment:
On Tue, Jan 13, 2015 at 8:33 AM, David Andrs <notifications@github.com>
wrote:

> In unit/src/MooseEnumTest.C
> <https://github.com/idaholab/moose/pull/4517#discussion-diff-22869292>:
>
> > @@ -130,6 +130,10 @@ MooseEnumTest::testErrors()
> >      std::string msg(e.what());
> >      CPPUNIT_ASSERT( msg.find("Invalid option") != std::string::npos );
> >    }
> > +  catch(...)
> > +  {
> > +    // even though we throw std::runtime_exception, we are not able to catch it here on Mac OS X (it has some other type)
> > +  }
>
> This might also be compiler specific (I did not try anything else than
> clang 3.5). I just wanted that test to pass on Mac OS X with our default
> compiler...
>
This isn't a TBB exception thing again, is it?

-- 
John